routes/tts_routes.py

```
@@ -0,0 +1,305 @@
+        fm[FM_AUDIO_DURATION] = result.duration_seconds
+    if result.format:
+        fm[FM_AUDIO_FORMAT] = result.format
+    if result.audio_id:
+        fm[FM_AUDIO_ID] = result.audio_id
+
+    save_ok, save_msg, _new_mtime = post_service.save_file(
+        article_path, body, frontmatter_data=fm, expected_mtime=expected_mtime
+    )
+
+    if not save_ok:
+        # 乐观锁冲突：save_msg 是一个 dict
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)